examples/override-example/variables.tf

```diff
@@ -0,0 +1,43 @@
+##############################################################################
+# Account Variables
+##############################################################################
+
+variable "ibmcloud_api_key" {
+  description = "The IBM Cloud platform API key needed to deploy IAM enabled resources."
+  type        = string
+  sensitive   = true
+}
+
+variable "prefix" {
+  description = "A unique identifier for resources. Must begin with a lowercase letter and end with a lowerccase letter or number. This prefix will be prepended to any resources provisioned by this template. Prefixes must be 16 or fewer characters."
+  type        = string
+  default     = "land-zone-vsi-qs"
+
+  validation {
+    error_message = "Prefix must begin with a lowercase letter and contain only lowercase letters, numbers, and - characters. Prefixes must end with a lowercase letter or number and be 16 or fewer characters."
+    condition     = can(regex("^([a-z]|[a-z][-a-z0-9]*[a-z0-9])$", var.prefix)) && length(var.prefix) <= 16
+  }
+}
+
+variable "region" {
+  description = "Region where VPC will be created. To find your VPC region, use `ibmcloud is regions` command to find available regions."
+  type        = string
+  default     = "us-south"
+}
+
+variable "resource_group" {
```

Comment:
Don't think this variable is used.

Reply:
Correct - coming from vsi quick start example. Fixed in both. We'll need to move to full checks eventually - currently using basics only https://github.com/terraform-ibm-modules/common-dev-assets/tree/main/module-assets/basic-pre-commit which do skip unused variable checks.